src/app/pokedex/components/MoveDisplay.tsx

```diff
@@ -5,7 +5,7 @@ import { getSignatureMoves } from "@/app/data/signatures";
 import { Move, moveCategories, MoveCategory } from "@/app/data/types/Move";
 import { Pokemon } from "@/app/data/types/Pokemon";
 import { isNull } from "@/app/data/util";
-import TypeBadge from "@/components/TypeBadge";
+import TypeBadge, { TypeBadgeElementEnum } from "@/components/TypeBadge";
```

Comment:
I've heard there are issues with Enums in typescript, so we might want to use another similar solution for this, but I should probably fact check that before actually acting on it.

Reply:
Haven't done that research yet, but to explicitly state what an alternate solution might be: Something like the "player"/"opponent" or "weak"/"strong" switches elsewhere in the code - basic strings constrained by union types. Alternatively, if we decide Enums are fine, we might want to switch those examples over, for consistency.

Reply:
OK, shockingly, it seems like the bad reputation of typescript enums were overblown clickbait. They're basically the same as the string union method under the hood, except that renaming the enum values is easier than with string constants. So I'm happy to keep this and potentially shift other examples over to enums in the future.